src/seedu/addressbook/data/AddressBook.java

```diff
@@ -143,4 +149,11 @@ public UniquePersonList getAllPersons() {
     public UniqueTagList getAllTags() {
         return new UniqueTagList(allTags);
     }
+    
+    public String getAllTaggings(){
+    	String entireList = "";
+    	for(Tagging values: this.allTaggings)
```

Comment:
Tagging values...sounds plural. Only in the for each loop this part is singular right?